packages/server/lib/hooks/connection/post-connection.ts

```diff
@@ -87,34 +100,7 @@ async function execute(createdConnection: RecentlyCreatedConnection, provider: s
                         : 'Unknown error';
 
                 const errorString = JSON.stringify(errorDetails);
-                const log = {
-                    level: 'error' as LogLevel,
-                    success: false,
-                    action: LogActionEnum.AUTH,
-                    start: Date.now(),
-                    end: Date.now(),
-                    timestamp: Date.now(),
-                    connection_id: upsertedConnection.connection_id,
-                    provider,
-                    provider_config_key: upsertedConnection.provider_config_key,
-                    environment_id: environment.id
-                };
-
-                const activityLogId = await createActivityLogAndLogMessage(log, {
-                    level: 'error',
-                    environment_id: environment.id,
-                    timestamp: Date.now(),
-                    content: `Post connection script failed with the error: ${errorString}`
-                });
```

Comment:
took the liberty to delete v1 logs, because I'm stripping everything down soon anyway